ci/setup-and-build.ps1

```diff
@@ -0,0 +1,202 @@
+    New-Item -Path "$($core_sdk_dir)\worker_sdk" -ItemType Directory -Force
+    New-Item -Path "$($binaries_dir)" -ItemType Directory -Force
+
+
+    Start-Event "download-spatial-packages" "build-unreal-gdk-:windows:"
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "tools", `
+        "schema_compiler-x86_64-win32", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\tools\schema_compiler-x86_64-win32.zip" `
+    )
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "schema", `
+        "standard_library", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\schema\standard_library.zip" `
+    )
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "worker_sdk", `
+        "c-dynamic-x86-msvc_md-win32", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86-msvc_md-win32.zip" `
+    )
+
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "worker_sdk", `
+        "c-dynamic-x86_64-msvc_md-win32", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip" `
+    )
+    Start-Process -Wait -PassThru -NoNewWindow -FilePath "spatial" -ArgumentList @(`
+        "package", `
+        "retrieve", `
+        "worker_sdk", `
+        "c-dynamic-x86_64-gcc_libstdcpp-linux", `
+        "$($pinned_core_sdk_version)", `
+        "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-gcc_libstdcpp-linux.zip" `
+    )
+    Finish-Event "download-spatial-packages" "build-unreal-gdk-:windows:"
+
+
+    Start-Event "extract-spatial-packages" "build-unreal-gdk-:windows:"
+    Expand-Archive -Path "$($core_sdk_dir)\tools\schema_compiler-x86_64-win32.zip" -DestinationPath "$($binaries_dir)\Programs\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\schema\standard_library.zip" -DestinationPath "$($binaries_dir)\Programs\schema\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86-msvc_md-win32.zip" -DestinationPath "$($binaries_dir)\Win32\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-msvc_md-win32.zip" -DestinationPath "$($binaries_dir)\Win64\" -Force
+    Expand-Archive -Path "$($core_sdk_dir)\worker_sdk\c-dynamic-x86_64-gcc_libstdcpp-linux.zip" -DestinationPath "$($binaries_dir)\Linux\" -Force
+    Finish-Event "extract-spatial-packages" "build-unreal-gdk-:windows:"
+
+
+    # Copy from binaries_dir
+    Copy-Item "$($binaries_dir)\Win64\include\*" "$($worker_sdk_dir)\" -Force -Recurse
+
+    Write-Log "Fetch MSBUILD_EXE location"
+    $msbuild_exe = "${env:ProgramFiles(x86)}\MSBuild\14.0\bin\MSBuild.exe"
```

Comment:
You could set the location of MSBuild.exe as a `param` on the script that defaults to this value.

That way, if someone has a custom installation location, they can override it on the command line and it will work.